src/components/ChatApp/MessageSection/MessageSection.react.js

```diff
@@ -585,6 +595,7 @@ class MessageSection extends Component {
               ThemeChangerComponents={components}
               bodyStyle={bodyStyle}
               actions={actions}
+              CustomSettingsDone={CustomSettingsDone}
```

Comment:
lets have the camel case syntax here too, `customSettingsDone` looks better. Rest looks good.